server.js

```diff
@@ -12,4 +12,124 @@ app.use(express.static(__dirname + '/client'));
 });*/
 app.listen(port, function () {
   console.log('Example app listening ' + port);
-});
\ No newline at end of file
+});
+// var config = require('./oauth.js');
+
+var passport = require('passport');
+var cookieParser = require('cookie-parser');
+app.use(cookieParser());
+// app.use(express.cookieParser());
+var bodyParser = require('body-parser');
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
+var session = require('express-session');
+app.use(passport.initialize())
+app.use(passport.session()); 
+
+
+// var config = require('./client/configs/oauth.js');
+var GoogleStrategy = require('passport-google-oauth2').Strategy;
+var dotenv = require('dotenv');
+dotenv.config();
+
+// create a user model
+
+var Schema = mongoose.Schema;
+
+var userSchema = new Schema({
+    oauthID: String,
+    name: String,
+    created: Date,
+    firstName: String,
+    lastName: String
+});
+
+var User = mongoose.model('User', userSchema);
+
+
+
+// config
+
+var GoogleStrategy = require('passport-google-oauth2').Strategy;
+
+passport.use(new GoogleStrategy({
+  // clientID: config.google.clientID,
+  // clientSecret: config.google.clientSecret,
+  // callbackURL: config.google.callbackURL
+  clientID: process.env.GOOGLE_CLIENT_ID,
+  clientSecret: process.env.GOOGLE_CLIENT_SECRET,
+  callbackURL: "http://localhost:3000/auth/google/callback"
```

Comment:
@slava89 this is local url. You should use one that is abstract. For heroku it should be like 'https://befriends.herokuapp.com/auth/google/callback'